apps/distgo/cmd/docker/build_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/palantir/godel/apps/distgo/cmd/dist"
 	"github.com/palantir/godel/apps/distgo/cmd/docker"
 	"github.com/palantir/godel/apps/distgo/params"
+	dockerparams "github.com/palantir/godel/apps/distgo/params/docker"
```

Comment:
Yeah I don't think that's great rationale -- I think the underlying issue here points to an inversion of control.

By the design of the package, it looks like both the `config` and `params` packages are meant to store objects/types that are read from config and used by other tasks, but shouldn't really house logic itself. For example, if you look at the `dist` params, the logic associated with them does not live in the `params` package.

I think this code should be updated/refactored to move the logic out of the `params` package to address the underlying dependency graph issue.